src/DefinitionStorage.php

```diff
@@ -211,7 +211,6 @@ private function isResolvable(string $id, array $building): bool
             }
         } finally {
             $this->buildStack += $building;
-            unset($building[$id]);
```

Comment:
The purpose was to clean up to free up memory. Why is it unused?

Reply:
`$building` don't use in next code